notebooks/keras-with-openvino-backend/keras-with-openvino-backend.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q keras keras-hub
</pre><p>openvino supported only for keras&gt;=3.8, I think it is better to specify lower bound explicitly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>